Agreements/Internal/TCCOW_TCNW.txt

```diff
@@ -11,6 +11,7 @@ COPX:*:*:SOPIT:EGLL:*:London TC COWLY:London TC NW:*:15000:SOPIT
 COPX:*:*:SOPIT:EGWU:*:London TC COWLY:London TC NW:*:15000:SOPIT5
 
 COPX:*:*:CLIPY:EGMC:*:London TC COWLY:London TC NW:*:15000:CLIPY
-COPX:*:*:CLIPY:EGSS:*:London TC COWLY:London TC NW:*:15000:CLIPY
```

Comment:
I would imagaine until we remove the archived procedures it is best to keep these agreements to ensure that people flying the old STARs will still show a COPX to TC COWLY